marlowe-playground-server/marlowe-playground-server.cabal

```diff
@@ -54,32 +67,33 @@ library
         http-client-tls -any,
         http-conduit -any,
         http-types -any,
-        playground-common -any,
         jwt -any,
         lens -any,
         marlowe -any,
-        marlowe-symbolic -any,
-        marlowe-actus -any,
         monad-logger -any,
         mtl -any,
         newtype-generics -any,
+        plutus-ledger-api -any,
         process -any,
         servant >= 0.16,
-        servant-client -any,
-        servant-client-core -any,
-        servant-purescript -any,
         servant-server -any,
-        stm -any,
-        temporary -any,
+        transformers -any,
         text -any,
+        temporary -any,
+        servant-client -any,
+        servant-purescript -any,
         time -any,
+        time-out -any,
         time-units -any,
-        transformers -any,
-        unordered-containers,
-        wai-cors -any,
-        web-ghc -any,
-        uuid -any,
-        validation -any
+        unordered-containers -any,
+        wai-cors -any
+        -- directory -any,
+        -- marlowe-symbolic -any,
+        -- marlowe-actus -any,
+        -- servant-client-core -any,
+        -- stm -any,
+        -- uuid -any,
+        -- validation -any
```

Comment:
Why leave these here? If they aren't needed, they don't need to be commented out.

Reply:
same